### DISCUSSION TITLE
"msg.payload.type is empty" when using the Injector node

### DISCUSSION THREAD

**Discussion by EpicLPer** [windkh/node-red-contrib-telegrambot#367] [General]
Heya,

I'm not sure what the issue is here and I'm a bit stuck now. I always get the error `msg.payload.type is empty` when I try to send a message via this:
![image](https://github.com/windkh/node-red-contrib-telegrambot/assets/8442196/be2b04c0-f225-41e0-b8fe-3d9027b55f57)

I also tried setting msg.payload.type to "message" in the function but this doesn't help either. Setting an empty JSON payload in the Injector won't work either, it simply throws `400 Bad Request: can't parse entities: Can't find end of the entity starting at byte offset 86 when processing message:`

Thanks for your help already!

**Comment by EpicLPer** [comment]
Well... just as I wrote this I figured it out.
Seems like when using the "Command" nodes it already has pre-filled JSON data which just gets handed off to the Sender node, which I had to manually supply to the Injector node (type and chatId) and now it works.

**Reply by windkh** [reply]
There are plenty of examples in the examples folder where you can see how different things can be done
